docs/metrics.rst

```diff
@@ -82,7 +82,9 @@ Name                                                Description
 ``executor.queued_tasks``                           Number of queued tasks on executor
 ``executor.running_tasks``                          Number of running tasks on executor
 ``pool.open_slots.<pool_name>``                     Number of open slots in the pool
+``pool.queued_slots.<pool_name>``                   Number of queued slots in the pool
 ``pool.used_slots.<pool_name>``                     Number of used slots in the pool
+``pool.occupied_slots.<pool_name>``                 Number of occupied slots in the pool
```

Comment:
What is the difference between used and occupied? Without futher info on seeing this doc I would guess they are the same.